src/modules/ROOT/pages/changelog/202303/08.json

```
@@ -0,0 +1,66 @@
+{
+  "meta": {
+    "releasedOn": "2023.10.18"
+  },
+
+  "changed": [
+    {
+      "what": [
+        "record import endpoints by replacing the old endpoints with new ones."
```

This doesn't describe the change at all; "we changed a thing by changing the thing".
The actual change isn't that exciting so it doesn't really matter but we can still do some tweaks in it.

```suggestion
        "the record import subsystem by upgrading it with the new data transformation framework."
```

This sounds cooler B)